app/src/main/java/com/example/cs567_3d_ui_project/qgis_map/QGisMapViewContainer.kt

```diff
@@ -17,6 +17,7 @@ import androidx.annotation.AttrRes
 import androidx.compose.ui.graphics.asAndroidBitmap
 import androidx.recyclerview.widget.GridLayoutManager
 import androidx.recyclerview.widget.RecyclerView
+import com.example.cs567_3d_ui_project.file_logging.LogFile
```

Comment:
Ah, sorry that you ended up going through all this qgis map stuff. I should've ended up deleting it cause we are not going to use it.